hpms/pom.xml

```diff
@@ -57,6 +74,12 @@
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
                 <version>${jacoco.version}</version>
+                <configuration>
+                    <excludes>
+                        <exclude>**/gov/cms/ab2d/hpms/hpmsapi/**</exclude>
```

Comment:
What's the reason for excluding these?

Reply:
Those are purely DTOs, nothing to test inside practically.

Reply:
Ok I think that makes sense for the DTOs, what about HPMSFetcherImpl though?

Reply:
That class is purely HTTP wiring and is mocked out in the test world.  We don't have MOCK HPMS server spun up anywhere in order to simulate what it would return.

Once HPMS is live, we can revisit and potentially add some integration smoke tests with them.